test/terraform_aws_ecs_example_test.go

```diff
@@ -0,0 +1,64 @@
+package test
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"terratest/modules/aws"
+	"terratest/modules/random"
+	"terratest/modules/terraform"
+	"testing"
+)
+
+// An example of how to test the Terraform module in examples/terraform-aws-ecs-example using Terratest.
+func TestTerraformAwsEcsExample(t *testing.T) {
+	t.Parallel()
+
+	expectedClusterName := fmt.Sprintf("terratest-aws-ecs-example-cluster-%s", random.UniqueId())
+	expectedServiceName := fmt.Sprintf("terratest-aws-ecs-example-service-%s", random.UniqueId())
+
+	// Pick a random AWS region to test in. This helps ensure your code works in all regions.
+	awsRegion := aws.GetRandomStableRegion(t, []string{"us-east-1", "eu-west-1"}, nil)
+
+	terraformOptions := &terraform.Options{
+		// The path to where our Terraform code is located
+		TerraformDir: "../examples/terraform-aws-ecs-example",
+
+		// Variables to pass to our Terraform code using -var options
+		Vars: map[string]interface{}{
+			"cluster_name": expectedClusterName,
+			"service_name": expectedServiceName,
+		},
+
+		// Environment variables to set when running Terraform
+		EnvVars: map[string]string{
+			"AWS_DEFAULT_REGION": awsRegion,
+		},
+	}
+
+	// At the end of the test, run `terraform destroy` to clean up any resources that were created
+	defer terraform.Destroy(t, terraformOptions)
+
+	// This will run `terraform init` and `terraform apply` and fail the test if there are any errors
+	terraform.InitAndApply(t, terraformOptions)
+
+	// Run `terraform output` to get the value of an output variable
+	taskDefinition := terraform.Output(t, terraformOptions, "task_definition")
+
+	// Look up the ECS cluster by name
+	cluster := aws.GetEcsCluster(t, awsRegion, expectedClusterName)
+
+	assert.Equal(t, int64(1), *cluster.ActiveServicesCount)
```

Comment:
It is safer to use `aws.Int64Value` and the like instead of directly dereferencing the pointers, as it handles the `nil` check for you. Can you update the file to use `aws.Int64Value` and its siblings wherever you dereference pointers?

Reply:
`github.com/aws/aws-sdk-go/aws` conflicts with `terratest/modules/aws`. I had to use an alias `awsSDK` if that's ok.

Reply:
Yup that is fine!